packages/utils/src/invariant.ts

```diff
@@ -0,0 +1,22 @@
+const isProduction: boolean = process.env.NODE_ENV === 'production';
+// @TODO: allow consumer to set the prefix e.g `[ Text.Body ]`
+const prefix: string = 'Invariant failed';
+
+// Throw an error if the condition fails
+// Strip out error messages for production
+// > Not providing an inline default argument for message as the result is smaller
+const invariant = (condition: any, message?: string): asserts condition => {
+  if (condition) {
+    return;
+  }
+  // GIVEN product, supress the message.
+  if (isProduction) {
+    return;
+  }
```

Comment:
Maybe we should log this instead of just ignoring it?

Reply:
Let's maybe ignore it for now and think about it. We could report to the console or sentry somehow.

Reply:
Why do you want to ignore it? I'm not sure I understand the intent here...

Reply:
> I think it's okay to first silence the error to fix the actual issue.

agree. let's get this out first, then we can refine it.

Reply:
I see, I was missing the context.